applications_tests/lethe-fluid/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ file(COPY lpbf_benchmark_mesh.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
 file(COPY lpbf_benchmark_mesh_3D.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
 file(COPY cylinder_close_outlet.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
 file(COPY jurins-law-2d-mesh-dimensioned.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
+file(COPY dg_tracer_sharp_edge_90_degrees_curve.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
```

Comment:
Can you add the .geo file to generate that mesh in the repository? It's always useful to keep the mesh generator.
If you made it with pointwise then it's ok, no need to add the pointwise file.